Lib/ssl.py

```diff
@@ -781,6 +781,12 @@ def verify_client_post_handshake(self):
         return self._sslobj.verify_client_post_handshake()
 
 
+def _sslcopydoc(func):
+    """Copy doc string from SSLObject to SSLSocket"""
```

Comment:
nitpick: it's called a "docstring" (no space), no?

Reply:
@tiran could you fix `doc string`? thanks